### DISCUSSION TITLE
Collision Tracking for Car

### DISCUSSION THREAD

**Discussion by anshp02** [microsoft/AirSim#4259] [Support Q&A]
I'm looking to track the collision state within AirSim and have it be written out to the output file that it generated by default with the recording feature. I've tried editing the CarPawnSimAPI.cpp file as outlined [here](https://microsoft.github.io/AirSim/modify_recording_data/) but have been unsuccessful at getting it to write out to the file. Any help with this would be much appreciated. Thanks!

**Comment by alonfaraj** [comment]
Can you share what you have tried so far?
Snippet or something might be a good start.
